Join all the instructions to enable logs in Jenkins (#11198)
DataDog/documentation: content/en/continuous_integration/setup_pipelines/jenkins.md

{{< img src="ci/ci-jenkins-plugin-config.png" alt="Datadog Plugin configuration for Jenkins"  style="width:100%;">}}

## Enable job log collection

This is an optional step that enables the collection of job logs. It involves two steps: enabling the job collection port on the Datadog Agent, and enabling job collection on the Datadog Plugin.

> ### Enable job log collection on the Agent

Reviewer: ```suggestion
### Datadog Agent
```

> First, the Datadog Agent needs to be configured to open a TCP port to collect logs:

Reviewer: ```suggestion
First, enable job log collection on the Datadog Agent by opening a TCP port to collect logs:
```

With this setup, the Agent listens in port `10518` for logs.

> ### Enable job log collection in the plugin

Reviewer: ```suggestion
### Datadog Plugin
```

> Second, enable job log collection on the plugin:

Reviewer: ```suggestion
Second, enable job log collection on the Datadog Plugin:
```
